### PR TITLE
MUSARK-959 - feature/uniform urls

### PR DESCRIPTION
**Pull request by ingarabr**
Frontend PR https://github.com/MUSIT-Norway/musit-frontend/pull/353

### CODE REVIEW

**Review by jarlah — COMMENTED** [review]
Not really convinced that its the right solution to continue support v1 urls. The best is to demand changes in the client

**Comment by kpmeen** [comment]
@jarlah we can't do that due to the Delphi client depending on the moveObject service. So until we know all installations of the KHM app has been updated with new URL, we have to have the v1 path in the config. The web client should not be allowed to use the v1 uri's though.

File: docker/dev/nginx/nginx.conf [hunk 1]
```diff
@@ -54,19 +54,31 @@ http {
...
+      proxy_pass http://thingaggregate:7070/service_thing_aggregate/;
     }
 
     location /api/actor/ {
-      proxy_pass http://actor:7070/;
+      proxy_pass http://actor:7070/service_actor/;
+    }
+
+    location /api/actor/v1/ {
+      proxy_pass http://actor:7070/service_actor/;
```
Comment (kpmeen) [inline docker/dev/nginx/nginx.conf]: Only the storagefacility service should have a v1 mapping enabled. The only reason is for the Delphi client not to break.

File: docker/dev/nginx/nginx.conf [hunk 2]
```diff
@@ -54,19 +54,31 @@ http {
     }
 
     location /api/geolocation/ {
-      proxy_pass http://geolocation:7070/;
+      proxy_pass http://geolocation:7070/service_geo_location/;
+    }
+
+    location /api/geolocation/v1/ {
+      proxy_pass http://geolocation:7070/service_geo_location/;
```
Comment (kpmeen) [inline docker/dev/nginx/nginx.conf]: See below comment

File: docker/utv/nginx/nginx.conf [hunk 3]
```diff
@@ -63,19 +63,31 @@ http {
     }
 
     location /api/geolocation/ {
-      proxy_pass http://geolocation:7070/;
+      proxy_pass http://geolocation:7070/service_geo_location/;
+    }
+
+    location /api/geolocation/v1/ {
+      proxy_pass http://geolocation:7070/service_geo_location/;
```
Comment (kpmeen) [inline docker/utv/nginx/nginx.conf]: See comment in dev nginx above

**Review by kpmeen — CHANGES_REQUESTED** [review]

**Comment by kpmeen** [comment]
@ingarabr please rebase and update the PR

**Review by steinaol — APPROVED** [review]
LGTM

**Review by kpmeen — APPROVED** [review]

**Pull request merged**